content/chainguard/administration/assumable-ids/identity-examples/aws-ec2-identity.md

```diff
@@ -0,0 +1,207 @@
+---
+title : "Create an Assumable Identity to Authenticate from an EC2 instance"
+linktitle: "AWS EC2"
+lead: ""
+description: "Procedural tutorial outlining how to create a Chainguard identity that can be assumed by an AWS role in EC2 instance."
+type: "article"
+date: 2025-03-14T08:48:45+00:00
+lastmod: 2025-03-31T08:48:45+00:00
+draft: false
+tags: ["PRODUCT", "PROCEDURAL", "CHAINGUARD IMAGES"]
```

Comment:
images -> containers. Is there a reason for the caps lock? If we want caps lock tags we should do that programmatically 

```suggestion
tags: ["PRODUCT", "PROCEDURAL", "CHAINGUARD Containers"]
```